sunpy/coordinates/frames.py

```diff
@@ -137,7 +137,45 @@ def __str__(self):
 
 
 @add_common_docstring(**_frame_parameters())
-class HeliographicStonyhurst(SunPyBaseCoordinateFrame):
+class BaseHeliographic(SunPyBaseCoordinateFrame):
+    """
+    Base class for HeliographicCarrington (HGC) and HeliographicStonyhurst (HGS) frames.
+    """
+    name = "base_heliographic"
```

Comment:
```suggestion
```
Don't give this class an explicit name.  It will have the name `baseheliographic` by default.